server/opts.go

```diff
@@ -3581,6 +3618,66 @@ func parseAllowedConnectionTypes(tk token, lt *token, mv interface{}, errors *[]
 	return m
 }
 
+// Helper function to parse auth callouts.
+func parseAuthCallout(mv interface{}, errors, warnings *[]error) (*AuthCallout, error) {
+	var (
+		tk token
+		lt token
+		ac = &AuthCallout{}
+	)
+	defer convertPanicToErrorList(&lt, errors)
+
+	tk, mv = unwrapValue(mv, &lt)
+	pm, ok := mv.(map[string]interface{})
+	if !ok {
+		return nil, &configErr{tk, fmt.Sprintf("Expected authorization callout to be a map/struct, got %+v", mv)}
+	}
+	for k, v := range pm {
+		tk, mv = unwrapValue(v, &lt)
+
+		switch strings.ToLower(k) {
+		case "issuer":
+			ac.Issuer = mv.(string)
```

Comment:
Might be safer for these various assertions to have `, ok =` otherwise a malformed response could result in a panic.

Reply:
See top of function, we capture panics and resolve. I think this was @wallyqs approach..